### ISSUE TITLE
move?

### ISSUE THREAD

**Issue by evanwill** [uidaholib/symposium-collection2019#1]
@owikle should I move this into the palouse digital org?

**Comment by owikle** [comment]
@evanwill Yes, please. I wanted to create it there but couldn't because I'm not part of the organization (can you add me, or is this the one that Devin needs to add me to? I can't remember). Thanks!

**Comment by evanwill** [comment]
@owikle I copied it to https://github.com/PalouseDH/symposium-collection-2019

Yeah, I can't add you to the org... but I added you to the repo
